Active_Record/app/models/gear.rb

```diff
@@ -0,0 +1,5 @@
+class Gear < ActiveRecord::Base
+  has_many :booking_gears, dependent: :destroy
+  has_many :bookings, through: :booking_gear
```

Comment:
you have no business logic in class `BookingGear`, so use `has_and_belongs_to_many`

Reply:
done